coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/HiveToRelConverter.java

```diff
@@ -92,12 +93,16 @@ protected SqlToRelConverter getSqlToRelConverter() {
 
   @Override
   protected SqlNode toSqlNode(String sql, Table hiveView) {
-    return parseTreeBuilder.process(trimParenthesis(sql), hiveView);
+    final SqlNode sqlNode = parseTreeBuilder.process(trimParenthesis(sql), hiveView);
+    if (hiveView != null) {
+      sqlNode.accept(new FuzzyUnionSqlRewriter(hiveView.getTableName(), this));
+    }
+    return sqlNode.accept(new HiveToCoralOperatorConverter(getSqlValidator()));
   }
 
   @Override
   protected RelNode standardizeRel(RelNode relNode) {
```

Comment:
can we clean up this method as well? or keep it and restructure it to do HiveToCoral conversions here?

Reply:
I think we can remove this method given we are migrating from RelNode transformation to SqlNode transformation. Will remove it.